pkg/secrethub/credentials/key.go

```diff
@@ -69,8 +79,16 @@ func ImportKey(credentialReader, passphraseReader Reader) (Key, error) {
 		return Key{}, err
 	}
 	if encoded.IsEncrypted() {
+		envPassphrase := os.Getenv("SECRETHUB_CREDENTIAL_PASSPHRASE")
+		if envPassphrase != "" {
+			credential, err := decryptKey([]byte(envPassphrase), encoded)
+			if err != nil {
+				return Key{}, err
```

Comment:
What error message does this display? Would it help the user to figure out what they should do to solve this error if we would add context indicating we're using a passphrase loaded from the `SECRETHUB_CREDENTIAL_PASSPHRASE` envvar?

Reply:
Well adding more context to the error would definitely help. I wrapped the error and mentioned the environment variable.